src/WorkItems/Logging/EventIds.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.WorkItems.Logging
+{
+    public static class EventIds
+    {
+        public static EventId LogsToProcessMetrics => new EventId(9001, nameof(LogsToProcessMetrics));
```

Comment:
>9001 [](start = 66, length = 4)

Why this value? Will we have a convention for assigning ids?

Reply:
We should come up with a convention. This was just an example. I don't expect it to survive when we add the real log messages. i.e. I don't think the LogToProcess metrics are interesting.

---
In reply to: [396832563](https://github.com/microsoft/sarif-sdk/pull/1820#discussion_r396832563) [](ancestors = 396832563)